src/web/src/components/Posts/Timeline.tsx

```diff
@@ -48,10 +49,25 @@ const Timeline = ({ pages, nextPage }: Props) => {
 
   // Iterate over all the pages (an array of arrays) and then convert all post
   // elements to <Post>
-  const postsTimeline = pages.map((page: Post[]): Array<ReactElement> | ReactElement =>
-    page.map(({ id, url }: Post) => <PostComponent postUrl={url} key={id} />)
-  );
 
+  let postsTimeline = new Array<ReactElement>();
+  {
+    let previousCount = pages[0].length;
+
+    for (let i = 0; i < pages.length; i++) {
```

Comment:
```suggestion
    for (let i = 0; i < pages.length; i+=1) {
```

Reply:
Yeah, I was suggested to fix the post count thing [here](https://github.com/Seneca-CDOT/telescope/pull/2531#discussion_r760472247)